src/k8s/requests/v2v/constants.js

```diff
@@ -7,3 +7,10 @@ export const CONVERSION_GENERATE_NAME = `${CONVERSION_BASE_NAME}-`;
 export const CONVERSION_PROGRESS_ANNOTATION = 'v2vConversionProgress';
 
 export const V2VVMWARE_DEPLOYMENT_NAME = 'v2v-vmware';
+
+export const VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAMESPACE = 'openshift'; // along the common-templates ...
+export const VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAME = V2VVMWARE_DEPLOYMENT_NAME;
```

Comment:
can we copy the value instead? this could be unintentionally changed when updating V2VVMWARE_DEPLOYMENT_NAME

Reply:
honestly, that was my intention. But I can split.

Reply:
done